### PR TITLE
Check CI status of a commit from build scripts

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers @oconnor663 @zanderz

(I went a little overboard on the script for this feature last night..)

The idea is that our release scripts will `git checkout` the keybase/client commit they're about to build and release, run this new `ci` command, and bail out if `ci`'s exit code was != 0:

screenshot from 2016-04-22 10-35-18

I'll follow up with PRs against the build scripts to perform the check.

Please review and let me know what you think, thanks!

### CODE REVIEW

File: ci.es6 [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+      console.error(logSymbols.error, chalk.red('CI tests failed'))
+      process.exit(2)
+      break
+    case 'pending':
+      console.error(logSymbols.warning, chalk.yellow('CI still pending'))
+      process.exit(3)
+      break
+    case 'success':
+    default:
+      if (status.statuses.length < 3) {
```
Comment (cjb) [inline ci.es6]: (This should be parameterized.)

File: ci.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env node
```
Comment (chrisnojima) [inline ci.js]: we might want to put these into src/lib dirs so its clear one is generated

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,21 @@
+The MIT License
```
Comment (chrisnojima) [inline LICENSE]: this isn't the license we use in client

**Comment by cjb** [comment]
@chrisnojima Switched to client's LICENSE and switched to src/ and lib/.

**Comment by chrisnojima** [comment]
👍

**Comment by zanderz** [comment]
LGTM, but FYI I had to do:
`npm install babel-polyfill --global
`
but then it worked:

```
c:\work\src\github.com\keybase\client>"\Program Files\nodejs"\ci
Checking keybase/client:bf0bd4798f

√ Your tests passed on CircleCI!
√ The Travis CI build passed
‼ The required number of tests weren't run (2 vs 3)

```

**Comment by chrisnojima** [comment]
actually lets add that as a dependency then 👍

**Comment by cjb** [comment]
Ah, it's a bummer that I have to ship babel to non-dev installs.  Added the dependency, thanks.

**Comment by chrisnojima** [comment]
this is a side effect of using things that need the polyfill. if you remove those deps then you don't need it

**Pull request merged**